vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc

```diff
@@ -57,8 +57,9 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - systemctl enable --now docker kubelet
       - if (curl -sk --max-time 10 https://{{ CLUSTER_APIENDPOINT_HOST }}:6443/healthz); then echo \"keepalived already running\";else systemctl start keepalived; fi
       - usermod -aG docker {{ IMAGE_USERNAME }}
-      - systemctl link /lib/systemd/system/monitor.keepalived.service 
-      - systemctl enable --now monitor.keepalived.service
+      - systemctl link /lib/systemd/system/monitor.keepalived.service
+      - systemctl enable monitor.keepalived.service
+      - systemctl start monitor.keepalived.service
```

Comment:
What is the reason to separate --now into enable+start?

Reply:
Earlier when the monitor.keepalived.service custom service was implemented, both were tried out and enable+start was working better.
Many problems were encountered during the testing process so possible issues needed to be ruled out one by one.